tasks.py

```diff
@@ -218,56 +169,11 @@ def update_terraform(c):
         )
 
 
-@task
-def format_disks(c, hosts="", disks=""):
-    """
-    Format disks with zfs, i.e.: inv format-disks --hosts build02 --disks /dev/nvme0n1,/dev/nvme1n1
-    """
-    for h in get_hosts(hosts):
-        _format_disks(h, disks.split(","))
-
-
-@task
-def setup_secret(c, hosts=""):
```

Comment:
Nixos-anywhere also has an option to provision secrets which would come handy for deploying ssh host keys.